### PR TITLE
issue-111: Add reward amount to the qr-code service

### PR DESCRIPTION
**Pull request by ValeriiLav**

### CODE REVIEW

File: svc/qrcodes/service.go [hunk 1]
```diff
@@ -54,11 +60,18 @@ func (s *Service) GetDataByQRCodeID(ctx context.Context, qrcodeID uuid.UUID) (in
 	if now.After(qrcodeData.ExpiresAt) {
 		return nil, ErrQRCodeExpired
 	}
+	if qrcodeData.RewardAmount.Float64 > 0 {
+		err := s.rs.AddDepositTransaction(ctx, userID, qrcodeData.RewardAmount.Float64, qrcodeID, "??")
```
Comment (dmitrymomot) [inline svc/qrcodes/service.go:64]: ?? = table name = `qrcodes`

**Review by dmitrymomot — CHANGES_REQUESTED** [review]

**Review by dmitrymomot — APPROVED** [review]

**Comment by dmitrymomot** [comment]
waiting for [SAT-121](https://nftprojectsboosty.atlassian.net/browse/SAT-121) will be completed

File: svc/qrcodes/service.go [hunk 2]
```diff
@@ -16,17 +16,23 @@ type (
...
 		pr qrcodeRepository
+		rs rewardsService
 	}
 
 	qrcodeRepository interface {
 		GetDataByQRCodeID(ctx context.Context, id uuid.UUID) (repository.Qrcode, error)
 	}
 
+	rewardsService interface {
+		AddDepositTransaction(ctx context.Context, userID uuid.UUID, amount float64, relationID uuid.UUID, relationType string) error
```
Comment (ValeriiLav) [inline svc/qrcodes/service.go:27]: AddDepositTransaction(ctx context.Context, userID, relationID uuid.UUID, relationType string, amount float64) error

File: svc/qrcodes/service.go [hunk 3]
```diff
@@ -54,11 +60,18 @@ func (s *Service) GetDataByQRCodeID(ctx context.Context, qrcodeID uuid.UUID) (in
 	if now.After(qrcodeData.ExpiresAt) {
 		return nil, ErrQRCodeExpired
 	}
+	if qrcodeData.RewardAmount.Float64 > 0 {
+		err := s.rs.AddDepositTransaction(ctx, userID, qrcodeID, "qrcodes", qrcodeData.RewardAmount.Float64)
```
Comment (ValeriiLav) [inline svc/qrcodes/service.go:64]: Move qrcodes to const.

Comment (ValeriiLav) [reply svc/qrcodes/service.go:64]: // RelationTypeQrcodes indicates that relation type "qrcodes".
	RelationTypeQrcodes = "qrcodes"

**Review by dmitrymomot — APPROVED** [review]

**Pull request merged**